lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../player.rb'
+
+module Players
+  class Human < Player
+
+    def move(board, delay = 0)
+      puts "What space would you like to move into, player #{self.token}?"
+      input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```